config/settings.yml

```
@@ -10,6 +10,7 @@ defaults: &defaults
   site_contact_username: ''
   site_contact_email: ''
   open_registrations: true
+  max_bio_chars: 160
```

If this is changed to 500, #42 can be closed.
```suggestion
  max_bio_chars: 500
```

---

Hold on I can update it...